Java/src/com/copilot/yagl4j/GamePadManager.java

```diff
@@ -6,47 +6,49 @@
  * @author Mark Lefering
  */
 public class GamePadManager {
-    static {
-        // Load yagl4j
-        System.loadLibrary("yagl4j");
-    }
 
-    /**
-     * Hooks a GamePadListener to the GamePadManager.
-     *
-     * @param gamePadListener The GamePadListener to hook to the GamePadManager.
-     * @return True if the listener is hooked, false if not.
-     */
-    public final native boolean hookGamePadListener(GamePadListener gamePadListener);
+  static {
+    // Load yagl4j
+    System.loadLibrary("yagl4j");
+  }
 
-    /**
-     * Unhooks a GamePadListener from the GamePadManager.
-     *
-     * @return True if the listener is unhooked, false if not.
-     */
-    public final native boolean unhookGamePadListener();
+  /**
+   * Hooks a GamePadListener to the GamePadManager.
+   *
+   * @param gamePadListener The GamePadListener to hook to the GamePadManager.
+   * @return True if the listener is hooked, false if not.
+   */
+  public final native boolean hookGamePadListener(
+      GamePadListener gamePadListener);
 
-    /**
-     * Starts the GamePadManager (starts the SDL thread in the native lib).
-     *
-     * @return True if started, false if not.
-     */
-    public final native boolean start();
+  /**
+   * Unhooks a GamePadListener from the GamePadManager.
+   *
+   * @return True if the listener is unhooked, false if not.
+   */
+  public final native boolean unhookGamePadListener();
 
-    /**
-     * Stops the GamePadManager (stops the SDL thread in the native lib).
-     *
-     * Note: waits for the the SDL thread to finish.
-     *
-     * @return True if stopped, false if not.
-     */
-    public final native boolean stop();
+  /**
+   * Starts the GamePadManager (starts the SDL thread in the native lib).
+   *
+   * @return True if started, false if not.
+   */
+  public final native boolean start();
 
-    /**
-     * Sets the game-pad mapping database.
-     *
-     * @param filePath The path to the database file.
-     * @return True if the database is loaded, false if not.
-     */
-    public final native boolean setGamePadMappingDatabase(String filePath);
+  /**
+   * Stops the GamePadManager (stops the SDL thread in the native lib).
+   *
+   * Note: waits for the the SDL thread to finish.
+   *
+   * @return True if stopped, false if not.
```

Comment:
Description could be clearer. E.g. true if successfully stopped. Now it reads as if it is possible it will keep running or smth

Reply:
Also `@code` should be used here. I'll create an issue.